docs/guides/qiskit-transpiler-service.mdx

```diff
@@ -77,12 +77,12 @@ The following examples demonstrate how to transpile circuits using the Qiskit Tr
     transpiled_circuit = cloud_transpiler_service.run(circuit)
     ```
 
-3. Produce a similar circuit and transpile it while letting the service to decide whether to use the AI-powered transpiling passes.
+3. Prodce a similar circuit and transpile it while letting the service to decide whether to use the AI-powered transpiling passes.
 
 
     ```python
     from qiskit.circuit.library import EfficientSU2
-    from qiskit_ibm_transpiler.transpiler_service import TranspilerService
+    from qiskit_ibm_transpiler.transpiler_service import TranspilerServic
```

Comment:
Correct the typo in 'TranspilerServic' to 'TranspilerService' for consistency with the codebase